controllers/common_test.go

```diff
@@ -61,24 +60,27 @@ var _ = Describe("Controller", func() {
 	}
 
 	grabK8sToken := func(g Gomega) string {
-		var secrets *corev1.SecretList
-
-		g.Eventually(func(gg Gomega) {
+		var result string
```

Comment:
What about just raising the default timeout in the makefile? IMHO These timeouts in code only confuse stuff unless they are expected.

Should be enough to set the `GOMEGA_DEFAULT_EVENTUALLY_TIMEOUT` env var in `make test` target in the Makefile.

Reply:
Note that we already do that in the github actions: https://github.com/redhat-appstudio/service-provider-integration-operator/blob/c705f789c5e2efd5a10f95eb71dabdfc072c89a3/.github/workflows/pr.yml#L109

Reply:
Yep, I suggested to place it in the Makefile in my first. I was just pointing out that we already do use it in the github action. But I agree with you, this should be set in the Makefile so that it is shared by all invocations.